openflow13/nxext_test.go

```diff
@@ -14,7 +15,7 @@ func TestNXActionResubmit(t *testing.T) {
 	action1 := NewNXActionResubmit(portID)
```

Comment:
I believe this test would benefit from using Go subtests (https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks) to isolate the different test actions from each other. Even better if you can make the test table-driven as well.